lib/MTT/Test/Build/Noop.pm

```diff
@@ -12,13 +12,24 @@ package MTT::Test::Build::Noop;
 
 use strict;
 
+use MTT::Values;
+
 #--------------------------------------------------------------------------
 
 sub Build {
+    my ($ini, $mpi_install, $config) = @_;
+
+    # In case of tests located in different folder and no much space available
+    # on target scratch, it can be useful to point another location of prebuilt
+    # tests.
+    my $test_bin_dir = Value($ini, $config->{full_section_name}, "test_bin_dir");
```

Comment:
I'm sorry to be picky, but the more I think about this, the more I think we should call it the "install" dir, because there's typically these generic directories:

* Source directory
* Build directory
* Install directory

The word "install" is a little better because the installed files may or may not be binaries (they'll almost certainly be executables of some flavor, but not necessarily binaries).

Reply:
Considering only these three options, it should be `install`. Should it be `external_installdir` or just `installdir` is OK?

Reply:
My $0.02: I think just `installdir` is fine -- no need for `external` in there.